x-pack/platform/plugins/shared/task_manager/server/task_store.ts

```diff
@@ -178,11 +177,6 @@ export class TaskStore {
       definitions: opts.definitions,
       allowReadingInvalidState: opts.allowReadingInvalidState,
     });
-    this.esClientWithoutRetries = opts.esClient.child({
-      // Timeouts are retried and make requests timeout after (requestTimeout * (1 + maxRetries))
-      // The poller doesn't need retry logic because it will try again at the next polling cycle
-      maxRetries: 0,
-    });
```

Comment:
Moving away from a child client in favour of setting `retryOnTimeout: false` on explicit requests.